app/src/main/AndroidManifest.xml

```diff
@@ -13,6 +13,7 @@
     android:roundIcon="@mipmap/ic_launcher_round"
     android:supportsRtl="true"
     android:theme="@style/OppiaTheme">
+    <activity android:name=".help.FAQ.FAQActivity"></activity>
```

Comment:
If you notice, we have written all  activities in Alphabetical order. So please rearrange this.

Reply:
Also the package inside `help` should be `faq` not `FAQ`